explorer/client/src/utils/numberUtil.ts

```diff
@@ -3,6 +3,8 @@
 
 // Number Suffix
 export const numberSuffix = (num: number): string => {
+    // Handle case where count not found:
+    if (num === 0) return '-';
```

Comment:
We should not do this in a general purpose util function. What if the caller just want to display "0 transactions"?

Reply:
Done